AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -334,9 +355,15 @@ class CallingContext: NSObject {
         guard let deviceManager = deviceManager else {
             return
         }
-        let camera = deviceManager.cameras[0]
-        localVideoStream = LocalVideoStream(camera: camera)
-        completionHandler(localVideoStream)
+        availableCameras[.front] = deviceManager.cameras.first(where: { $0.cameraFacing == .front })
+        availableCameras[.back] = deviceManager.cameras.first(where: { $0.cameraFacing == .back })
+
+        if let frontCamera = availableCameras[.front] {
```

Comment:
Do we need to force the localVideoStream to be the front Camera?

Reply:
Yes, it was front by default too. Previous implementation `deviceManager.cameras[0]` gets the front camera.